R/simpleCache.R

```diff
@@ -38,45 +38,51 @@ NULL
 #' environment variables you use in your instruction code. You can use this
 #' using the parameter buildEnvir (just provide a list of named variables).
 #' 
-#' @param cacheName	Unique name for the cache. Be careful.
-#' @param instruction  Quoted R code to be evaluated. The returned value of this
+#' @param cacheName	A character vector for a unique name for the cache. Be careful.
+#' @param instruction  R expression (bracketed) to be evaluated. The returned value of this
 #'     code is what will be cached under the cacheName.
-#' @param buildEnvir   You may choose to provide additional variables necessary
-#'     for evaluating the code in instruction.
-#' @param reload   forces re-loading the cache, even if it exists in the env.
-#' @param recreate forces reconstruction of the cache
-#' @param noload   noload is useful for: you want to create the caches, but not
-#'     load them if they aren't there (like a cache creation loop).
-#' @param cacheDir The directory where caches are saved (and loaded from).
-#'			Defaults to the global \code{\link[=setCacheDir]{RCACHE.DIR}} variable
-#' @param cacheSubDir You can specify a subdirectory within the \code{cacheDir}
-#' 			variable. Defaults to \code{NULL}.
-#' @param assignToVariable By default, \code{simpleCache} assigns the cache to a
+#' @param buildEnvir   An environment (or list) providing additional variables
+#'     necessary for evaluating the code in instruction.
+#' @param reload   Logical indicating whether to force re-loading the cache,
+#'     even if it exists in the env.
+#' @param recreate Logical indicating whether to force reconstruction of the
+#'     cache
+#' @param noload   Logical indicating whether to create but not load the cache.
+#'     noload is useful for: you want to create the caches, but not load (like a
+#'     cache creation loop).
+#' @param cacheDir Character vector specifying the directory where caches are
+#'			saved (and loaded from). Defaults to the variable set by
+#'			\code{\link[=setCacheDir]{setCacheDir()}}.
```

Comment:
What's the `=` do in the bracketed expression here?

Reply:
Dunno -- @vpnagraj ?

Reply:
@vreuter @nsheff that just helps construct the link to the setCacheDir function ... without it the link would point towards a package named setCacheDir (doesn't exist) rather than the function

Reply:
Ah OK. So it looks like this syntax mixes a couple of the ones in this [docs guide](http://r-pkgs.had.co.nz/man.html#text-formatting). Is this then doing something different from the options described there?

Reply:
yeah it's a mix of those ... this vignette has an example of the exact same linking: 

https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html#links

ironically in digging up that vignette to show you the example i realized i could have been using markdown the whole time :)